### PR TITLE
Build interface add project

### PR DESCRIPTION
**Pull request by tuanhai2712**
## Trello ticket URL
- Build interface add project

## Technical changes overview
- none

## SQL update
- none

## Other issues
- none

### CODE REVIEW

File: app/Http/Controllers/MemberController.php [hunk 1]
```diff
@@ -21,7 +21,8 @@ public function index()
 
     public function store(MemberValidatorRequest $request)
     {
-        return $this->member->store($request);
+        $store = $this->member->store($request);
```
[the remark this answers is not in the document]
Comment (jackdiff) [reply app/Http/Controllers/MemberController.php]: Why don't you return a json response ?

[the remark this answers is not in the document]
Comment (tuanhai2712) [reply app/Http/Controllers/MemberController.php]: i have fix code, and 'Why don't return a json': I have return a json response in Model Member

File: resources/assets/js/components/CreateProject.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    {
+        super(props);
+        this.state = {
+            project_name: '',
+            project_info: '',
+            project_deadline: '',
+            project_type: '',
+            project_status: ''
+        };
+        this.handleChange1 = this.handleChange1.bind(this);
```
[the remark this answers is not in the document]
Comment (tuanhai2712) [reply resources/assets/js/components/CreateProject.js]: I need to use this (binding) to dump the data into the function setState if I do not use it I can not use function setState

File: resources/assets/js/components/CreateProject.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    {
+        super(props);
+        this.state = {
+            project_name: '',
+            project_info: '',
+            project_deadline: '',
+            project_type: '',
+            project_status: ''
+        };
+        this.handleChangeName = this.handleChangeName.bind(this);
```
Comment (jackdiff) [inline resources/assets/js/components/CreateProject.js]: Remove these bind(this).

**Pull request merged**